File: 1. Begginers/1037 Interval.py
n = float( input( ))

if n>=0 and n<=100:
    if   True and n<=25:
        print('Intervalo [0,25]')
    elif n>25 and n<=50:
        print('Intervalo (25,50]')
    elif n>50 and n<=75:
        print('Intervalo (50,75]')
    elif n>75 and  True:
        print('Intervalo (75,100]')
else:
    print( 'Fora de intervalo' )

# Time: 0.016s

File: 1. Begginers/1037-Interval.cpp
#include <iostream>
using namespace std;

float num;

main(){
    cin >> num;

      if ((num >= 0) and (num <=  25)){
        cout<< "Intervalo [0,25]"<< endl;
    }
    else if ((num > 25) and (num <=  50)){
        cout<< "Intervalo (25,50]"<< endl;
    }
    else if ((num > 50) and (num <=  75)){
        cout<< "Intervalo (50,75]"<< endl;
    }
    else if ((num > 75) and (num <= 100)){
        cout<< "Intervalo (75,100]"<< endl;
    }
    else{
        cout<< "Fora de intervalo"<< endl;
    }
}
